Extensions/tweaks.js

```diff
@@ -678,7 +678,9 @@ XKit.extensions.tweaks = new Object({
 		}
 
 		if (XKit.extensions.tweaks.preferences.hide_share_menu.value) {
-			XKit.tools.add_css(".post_control.share { display: none; } ", "xkit_tweaks_hide_share");
```

Comment:
this should be retained for/on non-react pages

Reply:
Oh no! I'll have to rerecord the code-writing part of the tutorial I'm working on. Thanks for pointing this out!